applications/base/skills.py

```
@@ -84,21 +88,20 @@ class get_pose_implementation(BaseTool):
     return_direct: ClassVar[bool] = False
     verbose: ClassVar[bool] = False
 
-    def _run(self, robot_to_use: int = 1, wait: bool = True) -> list:
+    def _run(self, robot_to_use: int = 1, wait: bool = True, tool_num: int = 0, user_num: int = 0) -> Optional[list]:
```

Ruff (0.12.2)

Unused method argument: `wait`

(ARG002)